llvm/test/CodeGen/X86/atomic-load-store.ll

```diff
@@ -1,12 +1,18 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
 ; RUN: llc < %s -mtriple=x86_64-apple-macosx10.7.0 -verify-machineinstrs | FileCheck %s
 ; RUN: llc < %s -mtriple=x86_64-apple-macosx10.7.0 -verify-machineinstrs -O0 | FileCheck %s
+; RUN: llc %s --mtriple=x86_64 -o - | FileCheck %s --check-prefix=CHECK0
 
```

Comment:
This seems like a good preexisting test to add the <1 x n> cases to. It has a specific triple `apple`, and when generalizing it, the difference in the output tests is not much, and so it duplicates quite a bit. However, I do notice that bfloat passes only when `--mtriple=x86_64`, so we may want to add the additional run line.

Reply:
Just copy the mtriple that's already here, the OS should not be significant for the purpose here